source/UF/CoconesofSpans.lagda

```diff
@@ -4,6 +4,8 @@ We will prove some results about cocones of spans. For details see section 23
 of Introduction to Homotopy Type Theory by Egbert Rijke (HoTTest summer school
 version:
 https://github.com/martinescardo/HoTTEST-Summer-School/blob/main/HoTT/hott-intro.pdf)
+and the development found in the Agda Uni-Math library
```

Comment:
I don't think there is a hyphen in "Uni-Math". Also they write "agda-unimath" (see e.g. the page https://unimath.github.io/agda-unimath/).

Reply:
I did not base any code on anything from the library but I did look at the library for inspiration at the beginning of the project.

Reply:
Oh I see. In the pushout file I do say that it is inspired but I simply point to the library in the cocone file. I will fix this.